tasks/systemd_setup.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: install the systemd unit file
+  template:
+    src: 'etc/systemd/system/tinc@.service.j2'
+    dest: '/etc/systemd/system/tinc@.service'
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
+
+- name: autostart tinc
+  service:
+    name: 'tinc@{{ tinc_network }}'
+    enabled: 'yes'
+    state: 'started'
```

Comment:
Then perhaps adding a fact that detects local init system would be a good idea after all... I tried to avoid this, but you got me a use case (`service` module with instanced units) for it. I'll add that soon, then issue with `ignore_errors: True` can be solved that way.